.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+### Type of Change
+
+Check all that apply:
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+- [ ] migration required (a model change requiring a mongodb script or migration script)
+- [ ] UI change
+
+## Screenshots
+
+Please provide screenshots / animations for any change that involves the UI.  Please provide animations to demonstrate user interaction / behavior changes
+
+## How Has This Been Tested?
+
+Please describe the manual tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+## Checklist:
+
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have added tests that prove my fix is effective or that my feature works
```

Comment:
Since we're trying to ensure good commit messages before merging, I wonder if another checkbox here about reviewing the final title/desc of the PR would be helpful?